src/QirRuntime/lib/QIR/bridge-qis.ll

```diff
@@ -29,122 +29,264 @@
 %struct.QirCallable = type opaque
 %struct.QirRange = type { i64, i64, i64 }
 %struct.QirString = type opaque
+;%PauliId = type {i32}

```

Comment:
Forgot to remove or forgot to uncomment or is it just a clarification?

Reply:
The intent was to clarify... Does it help?

Reply:
May be 
```llvm
; Clarifications:
;%PauliId = type {i32}
```
?
(to avoid making an impression that it is forgotten to be either removed or uncommented)